### PR TITLE
styling adjusted

### PR DESCRIPTION
**Pull request by Sergei-Nikolaienko**
https://Sergei-Nikolaienko.github.io/layout_catalog/
https://Sergei-Nikolaienko.github.io/layout_catalog/report/html_report/

### CODE REVIEW

**Comment by Sergei-Nikolaienko** [comment]
NPM won't run for some reason - was asked to create a pull request anyway to check the finished task.

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Frontend practice with catalog page
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_catalog/report/html_report/)
+- [DEMO LINK](https://Sergei-Nikolaienko>.github.io/layout_catalog/)
```
Comment (Viktor-Kost) [inline readme.md:3]: Here you have typos. Use Sergei-Nikolaienko instead of Sergei-Nikolaienko>

File: src/styles/utils/normalize.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+body,
+ul,
+h2,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  list-style: none;
```
Comment (Viktor-Kost) [inline src/styles/utils/normalize.scss:18]: This rule is for `ul`, so then you will need to use this rule again for example here 
```
.nav {
  margin: 0;
  padding: 0;
  display: flex;
  list-style: none;
}
```

File: src/styles/blocks/stars.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin: 0;
+}
+
+.stars--1 div:nth-child(-n+1),
```
Comment (Viktor-Kost) [inline src/styles/blocks/stars.scss:17]: Here, just as recommendation, we can use a sass loop [link](https://sass-lang.com/documentation/at-rules/control/for)

File: src/styles/blocks/header.scss [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  content: " ";
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  bottom: 0;
+  background: var(--primary-blue);
+  position: absolute;
+}
+
+.nav a:hover {
```
Comment (Viktor-Kost) [inline src/styles/blocks/header.scss:56]: Please, use style selectors instead of tag one

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Well done!
Just apply some recommendations to improve your code

**Comment by Sergei-Nikolaienko** [comment]
> 
image
have deployed, not sure why it won't show up

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Good job!

And, please, check your test report, cause you have some diffs there

**Review by Viktor-Kost — COMMENTED** [review]
Have you updated a link for tests, cause there is still a link with failed test?

File: src/index.html [hunk 5]
```diff
@@ -1,12 +1,325 @@
...
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <a class="logo" href="#">
+        <img src="./images/logo.png" alt="Store Logo">
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__list">
+            <a class="nav__item nav__item--is--active is-active" href="#">Apple</a>
```
Comment (OlEzhA0) [inline src/index.html:24]: you don't use class `is-active`.

also with BEM convention we should give name for modifier with one dash
```suggestion
            <a class="nav__item nav__item--is-active" href="#">Apple</a>
```

use this rule for future
`block-name__elem-name--mod-name--mod-val`

File: src/index.html [hunk 6]
```diff
@@ -1,12 +1,325 @@
...
+
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <a class="logo" href="#">
+        <img src="./images/logo.png" alt="Store Logo">
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__list">
```
Comment (OlEzhA0) [inline src/index.html:23]: I'm not sure that is good class name for element
each tag `li` is item, not a list

you can use class `nav__list` for `ul` tag

File: src/index.html [hunk 7]
```diff
@@ -1,12 +1,325 @@
...
+            <a class="nav__item" href="#">Smartphones</a>
+          </li>
+          <li class="nav__list">
+            <a 
+              class="nav__item" 
+              data-qa="nav-hover" 
+              href="#"
+            >Laptops & Computers
+            </a>
```
Comment (OlEzhA0) [inline src/index.html:38]:
```suggestion
            <a 
              class="nav__item" 
              data-qa="nav-hover" 
              href="#"
            >
              Laptops & Computers
            </a>
```

File: src/index.html [hunk 8]
```diff
@@ -1,12 +1,325 @@
...
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+            </div>
+            
+            <p class="card__reviews-no">Reviews: 5</p>
```
Comment (OlEzhA0) [inline src/index.html:83]: what do you mean `card__reviews-no`?

File: src/index.html [hunk 9]
```diff
@@ -1,12 +1,325 @@
...
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <h3 class="card__price-value">$2,199</h3>
+          </div>
+          <a 
+            class="card__buy-cta" 
+            href="#" 
+            data-qa="card-hover"
+          >Buy
+          </a>
```
Comment (OlEzhA0) [inline src/index.html:94]:
```suggestion
          <a 
            class="card__buy-cta" 
            href="#" 
            data-qa="card-hover"
          >
            Buy
          </a>
```

File: src/styles/blocks/card.scss [hunk 10]
```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $cards-width;
+  background: white;
+  border: 1px solid #f3f3f3;
```
Comment (OlEzhA0) [inline src/styles/blocks/card.scss:6]: add this color to variables

File: src/styles/blocks/card.scss [hunk 11]
```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $cards-width;
+  background: white;
```
Comment (OlEzhA0) [inline src/styles/blocks/card.scss:5]: use the sixteen Named colors `#fff` and you can also put it to vars

File: src/styles/blocks/card.scss [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+    margin: 0 auto 40px;
+  }
+
+  &__product-name {
+    color: var(--primary-grey);
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
```
Comment (OlEzhA0) [inline src/styles/blocks/card.scss:26]: you can try to use `mixins` to create some typography styles, put there color, font-size, line-height, font-weight

File: src/styles/blocks/header.scss [hunk 13]
```diff
@@ -0,0 +1,58 @@
+header {
+  display: flex;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
```
Comment (OlEzhA0) [inline src/styles/blocks/header.scss:7]: add class for header

File: src/styles/blocks/header.scss [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
```
Comment (OlEzhA0) [inline src/styles/blocks/header.scss:12]: it can be `header__logo` etc

File: src/styles/blocks/header.scss [hunk 15]
```diff
@@ -0,0 +1,58 @@
...
+
+.nav {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+
+  &__list {
+    padding-left: 20px;
```
Comment (OlEzhA0) [inline src/styles/blocks/header.scss:21]: add spaces for other elements with margin

File: src/styles/utils/_variables.scss [hunk 16]
```diff
@@ -0,0 +1,9 @@
+$cards-gap: 48px;
+$cards-width: 200px;
+$catalog-width: 4 * $cards-width + 3 * $cards-gap;
```
Comment (OlEzhA0) [inline src/styles/utils/_variables.scss:3]: what is 4 and 3? mb create constants for them

File: src/styles/utils/_variables.scss [hunk 17]
```diff
@@ -0,0 +1,9 @@
...
+
+:root {
+  --primary-blue: #00acdc;
+  --primary-grey: #060b35;
+  --secondary-grey: #616070;
+}
```
Comment (OlEzhA0) [inline src/styles/utils/_variables.scss:9]: 👍🏼

File: src/styles/blocks/header.scss [hunk 18]
```diff
@@ -0,0 +1,58 @@
...
+  
+  &__list:first-of-type {
+    padding: 0;
+  }
+  
+  &__item {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-decoration: none;
```
Comment (OlEzhA0) [inline src/styles/blocks/header.scss:32]: you reseted this style for all tags `a`

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
Good job! Let's resolve issues and take a look why some tests failed

**Review by Virass — APPROVED** [review]

**Pull request closed without merging**